spec/shr_procedure.txt

```diff
@@ -15,7 +16,7 @@ Description:	"A procedure that has been performed."
 0..1			ReasonCode
 0..1			ReasonReference
 0..0			Method	// Procedure doesn't have Method		
-//0..*			InputFinding
+0..*			TreatmentIntent
```

Comment:
@markkramerus - Can we just call this "ProcedureIntent"?  Not all procedures are treatments.